RecoPPS/Local/test/totemT2NewDigi_reco_cfg.py

```diff
@@ -0,0 +1,71 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('RECO')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.MessageLogger.cerr.threshold = "DEBUG"
+# enable LogDebug messages only for specific modules
+process.MessageLogger.debugModules = ["Totem"]
+
+process.load('Configuration.EventContent.EventContent_cff')
+#process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+#from Configuration.AlCa.GlobalTag import GlobalTag
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
+
+# raw data source
+process.source = cms.Source("NewEventStreamFileReader",
+    fileNames = cms.untracked.vstring(
+#        '/store/t0streamer/Data/Physics/000/286/591/run286591_ls0521_streamPhysics_StorageManager.dat',
+#        'file:../../../../run364538_ls0001_streamA_StorageManager.dat',
+        'file:../../../../../../CMSSW_13_1_0_pre1/t2v2p1/run364983_ls0001_streamA_StorageManager.dat',
```

Comment:
This wont work in CMSSW, please move these files somewhere publically visible at least. Would be also nice to have a unit test running this cfg file